press/legacy_publishing/litezip.py

```diff
@@ -67,8 +61,8 @@ def publish_litezip(struct, submission, db_conn):
         except Unchanged:
             pass  # only publish content that has changed.
 
-    any_changes = id_map != {}
-    if any_changes:
+    collxml_changed = bool(id_map)
+    if id_map:
```

Comment:
And if you rename the line above, you can use it here. 
`if modules_changed:  # update the collection`